core/controllers/editor_test.py

```diff
@@ -261,169 +261,6 @@ def _put_and_expect_400_error(payload):
         self.logout()
 
 
-class ExplorationEditorLogoutTest(BaseEditorControllerTests):
```

Comment:
You might as well keep one test that checks the intended behaviour.

Reply:
Done.